prober/prober.go

```diff
@@ -180,7 +180,7 @@ func (p *Prober) checkOneProbe(ctx context.Context, probe Probe, res chan *Probe
 	res <- probeResult
 }
 
-func ProberWrapper(probe Probe, res chan error) {
+func Wrapper(ctx context.Context, probe Probe, res chan error) {
```

Comment:
```suggestion
//nolint:revive
func ProberWrapper(ctx context.Context, probe Probe, res chan error) {
```

Following ths discussion